### DISCUSSION TITLE
Allow searching recipes by those that are not assigned any category or tag

### DISCUSSION THREAD

**Discussion by ryan408** [mealie-recipes/mealie#4626] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/mealie-recipes/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

I have entered all my recipes into Mealie and am now going through assigning each to categories I have created (Entree, Appetizer, etc.).  There is no way currently to view which ones have a category assigned and which do not without clicking into each recipe, which is very time consuming.  As I add new recipes in a hurry, I'd like to be able to go back later and use the advanced search to not only find recipes assigned to a category, but also recipes that are not yet assigned to any category.  

### Please provide a concise description of the feature that would resolve your issue.

There is currently a toggle in the advanced search for categories that says "Has any".  A similar toggle or option to search "Has none" would allow me to select all categories and see which recipes have none of the categories assigned.  Then I can work through those recipes and assign categories to each.

### Please consider and list out some caveats or tradeoffs made in your design decision

I'm not sure if there is an impactful tradeoff.  Just one more option to essentially anti-join on the list of selected categories.

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by DanielBurns** [comment]
This would be so helpful for exactly the reasons you state. I hope this is recognized and implemented.

**Comment by michael-genson** [comment]
You can sort by category or tag in the recipe data management page. The sort order doesn't mean much, but it will group recipes with zero categories/tags together at the top (or bottom, depending on sort direction)

**Reply by DanielBurns** [reply]
that's a good tip. thank you.

**Comment by ryan408** [comment]
Yeah, this could work. I just tried it and it’s sufficient. Thanks for the tip.
________________________________

**Comment by leitenky** [comment]
Yes i was not aware of that feature that seams like it will work. its not as intuitive as I was hoping though. Thanks for the tip though!

**Comment by ChipSkylark37** [comment]
Reading both this and another discussion: https://github.com/mealie-recipes/mealie/discussions/4701 

Though the 'Data Management' method works, I think this opens the conversation on a more 'Advanced Search & Filtering' for Mealie as a whole. I will be opening a new Feature Request for this, and linking to a few other thread, with this being one of them. 

There's generally ways to do a lot of what we'd like, just in different spots. 

I will update once I create the new thread.
